sandbox/apparmor/abi.go

```diff
@@ -0,0 +1,103 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package apparmor
+
+// Abi30Text is contents of the /etc/apparmor.d/abi/3.0 file.
+//
+// This file defines the apparmor ABI at version 3.0.
+const Abi30Text = `
```

Comment:
Why cap A if we do not use outside package apparmor?

Reply:
No real preference, it's a constant so it felt okay. It can be private, only the path needs to be public.